src/wallet/wallet.cpp

```diff
@@ -2112,53 +2112,38 @@ bool CWalletTx::IsEquivalentTo(const CWalletTx& _tx) const
         return CTransaction(tx1) == CTransaction(tx2);
 }
 
-std::vector<uint256> CWallet::ResendWalletTransactionsBefore(interfaces::Chain::Lock& locked_chain, int64_t nTime)
```

Comment:
I'm likely missing something, but should `ResendWalletTransactionsBefore` be removed from `/src/wallet/wallet.h:950-951` as well?

```cpp
    // ResendWalletTransactionsBefore may only be called if fBroadcastTransactions!
    std::vector<uint256> ResendWalletTransactionsBefore(interfaces::Chain::Lock& locked_chain, int64_t nTime);
```

Reply:
You're not missing anything. Now fixed. Thanks!